File: BehaviorPattern/Command/Python/Concretes/LightOnCommand.py

from Interfaces import Command
from Receivers import Light

class LightOnCommand(Command):
    """
        Class Light On Command implementation Command Interface
    """
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.light = kwargs.get('light', None)
        if args:
            self.light = args[0]
            
    # Override
    def execute(self):
        """
            Execute command
        """
        self.light.on()

File: BehaviorPattern/Command/Golang/Concretes/LightOnCommand.go

package concretes

import (
	"v/Receivers"
	"v/Interfaces"
)

/**
 * Class Light On Command implementation Command Interface
 */
type LightOnCommand struct {
	Light receivers.Light
}

// Constructor
func NewLightOnCommand(light receivers.Light) interfaces.Command {
	return &LightOnCommand{Light: light}
}

//  Override Execute command
func (l *LightOnCommand) Execute() {
	l.Light.Off()
}